### PR TITLE
Integrating GPTRouter

### PR DESCRIPTION
**Pull request by sirjan-ws-ext**
**Description:** Adding a langchain integration for [GPTRouter](https://gpt-router.writesonic.com/) 🚀 , 
 **Tag maintainer:** @Gupta-Anubhav12 @samanyougarg @sirjan-ws-ext  
 **Twitter handle:** [@SamanyouGarg](https://twitter.com/SamanyouGarg)
 
Integration Tests Passing:
Screenshot 2023-12-19 at 5 45 31 PM

### CODE REVIEW

File: libs/langchain/langchain/chat_models/gpt_router.py
Comment (baskaryan) [inline libs/langchain/langchain/chat_models/gpt_router.py:1]: no need to add re-exports of new community integrations in langchain (existing ones are there for backwards compatibility, but generally we're moving towards decoupling langchain and langchain-community)

Comment (sirjan-ws-ext) [reply libs/langchain/langchain/chat_models/gpt_router.py:1]: @baskaryan Got it, PTAL now

**Comment by sirjan-ws-ext** [comment]
@baskaryan Moved the integration to `langchain_community` now, PTAL :)

File: libs/langchain/langchain/chat_models/gpt_router.py
Comment (sirjan-ws-ext) [reply libs/langchain/langchain/chat_models/gpt_router.py:1]: lmk if it looks good now

**Pull request merged**